src/components/CIPipelineN/TaskTypeDetailComponent.tsx

```diff
@@ -207,28 +210,45 @@ export function TaskTypeDetailComponent() {
     }
 
     const ValueContainer = (props) => {
-        let value = props.getValue()[0]?.label
+        const value = props.selectProps?.value?.value
         return (
             <components.ValueContainer {...props}>
                 <>
                     {!props.selectProps.menuIsOpen &&
-                        (value ? `${value}` : <span className="cn-5">Select or enter image</span>)}
+                        (value ? (
+                            <div className="cn-7 fs-12 flex left">{value}</div>
+                        ) : (
+                            <span className="cn-5">Select or enter image</span>
+                        ))}
                     {React.cloneElement(props.children[1])}
                 </>
             </components.ValueContainer>
         )
     }
 
+    const MenuList = (props) => {
```

Comment:
move this to utils as well

Reply:
There is already a ValueContainer present in utils , so it is creating a error.